docs/validator.md

```diff
@@ -14,7 +14,7 @@ As of Dec 11th 2023, the expected cost number of DataItems queried via Apify is
 
 # System Requirements
 
-Validators do not require a GPU and should be able to run on a relatively low-tier machine, as long as it has sufficient network bandwidth and disk space.
+Validators require at least 32 GB of RAM but do not require a GPU and should be able to run on a relatively low-tier machine otherwise, as long as it has sufficient network bandwidth and disk space.
```

Comment:
I think we should recommend a CPU with a few cores and remove the disk space note.

Reply:
Updated.